cs/src/Contracts/Tunnel.cs

```diff
@@ -127,6 +127,12 @@ public Tunnel()
     [MaxLength(TunnelMaxPorts)]
     public TunnelPort[]? Ports { get; set; }
 
+    /// <summary>
```

Comment:
Why set it on the tunnel object and not on `TunnelOptions`? Tunnel options also apply to tunnel ports, and so can be controlled with more granularity.

Reply:
Should this be inside of `TunnelPort`?
Or you're thinking that inspection should be enabled on a Tunnel and they all Ports on that tunnel get auto-inspected?

Reply:
The tunnel options are stored as part of the tunnel and tunnel port objects. They're not a separate object.